dashboard/models/models.py

```diff
@@ -2055,6 +2058,12 @@ class StudySite(db.Model):
     code = db.Column('code', db.String(32))
     download_script = db.Column('download_script', db.String(128))
     post_download_script = db.Column('post_download_script', db.String(128))
+    xnat_url = db.Column('xnat_url', db.String(128))
+    xnat_archive = db.Column('xnat_archive', db.String(32))
+    xnat_convention = db.Column(
```

Comment:
Are multiple conventions still relevant? Or has everything been moved over to the "KCNI" convention now?

Reply:
I left that in there in case the convention ever changes again. No need to strip it out and assume one single convention when things could change again later